<ui version="4.0" >
 <class>EditDirListDialog</class>
 <widget class="QDialog" name="EditDirListDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Choose Directories</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item rowspan="6" row="0" column="0" >
      <widget class="QListView" name="dirList" >
       <property name="editTriggers" >
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QToolButton" name="buttonOpen" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QToolButton" name="buttonAdd" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="ggap.qrc" >
         <normaloff>:/icons16/list-add.png</normaloff>:/icons16/list-add.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QToolButton" name="buttonRemove" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="ggap.qrc" >
         <normaloff>:/icons16/list-remove.png</normaloff>:/icons16/list-remove.png</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QToolButton" name="buttonUp" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="ggap.qrc" >
         <normaloff>:/icons16/go-up.png</normaloff>:/icons16/go-up.png</iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="1" >
      <widget class="QToolButton" name="buttonDown" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="ggap.qrc" >
         <normaloff>:/icons16/go-down.png</normaloff>:/icons16/go-down.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dirList</tabstop>
  <tabstop>buttonOpen</tabstop>
  <tabstop>buttonAdd</tabstop>
  <tabstop>buttonRemove</tabstop>
  <tabstop>buttonUp</tabstop>
  <tabstop>buttonDown</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="ggap.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditDirListDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditDirListDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
